source/cppfront.cpp

```diff
@@ -3525,7 +3525,22 @@ class cppfront
                     emit(*x.op);
                 }
                 printer.print_cpp2(" ", n.position());
-                emit(*x.expr);
+                //  When assigning a single expression-list, we can
+                //  take over direct control of emitting it without needing to
+                //  go through the whole grammar, and surround it with braces
+                if (
+                    x.op->type() == lexeme::Assignment
+                    && x.expr->is_expression_list()
```

Comment:
This looks good, but it needs

                    && x.expr->get_expression_list()->expressions.size() > 1

otherwise `x = (move y);` will grow braces we don't need. I'll make the change directly and merge.

Thanks!

Reply:
On second thought, I was wrong -- we want those `{` `}` in other cases. Will revert my changes...